README.md

```diff
@@ -53,7 +53,8 @@ function MyApp() {
 |activeBboxIndex|Index of active bbox from bboxes array|n/a| `0`                                                                                                                                                                                                                                                                                                               |
 |activeBboxId|Id of active tree bbox|n/a| `"0:3:11:0"`                                                                                                                                                                                                                                                                                                               |
 |showAllPages|Boolean flag to show pdf with all pages with scroll or single page|false| `true`                                                                                                                                                                                                                                                                     
-|isTreeBboxesVisible|Boolean flag to show tree bboxes|false| `true`                                              
+|isTreeBboxesVisible|Boolean flag to show tree bboxes|false| `true`  
+|treeBboxSelectionMode|String that determines how tree bboxes are highlighted:<ul><li>The value "current" selects only the current bbox</li><li>The value "all" selects the current bbox and all child bboxes</li></ul> |current| One of the following values: <ul><li>`"current"`</li><li>`"all"`</li></ul>                                                                                  
```

Comment:
Let's use more meaning full names for the modes, e.g. "SELECTED" and "SELECTED_WITH_NESTED"/"SELECTED_WITH_KIDS"